about.html

```diff
@@ -0,0 +1,200 @@
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-light fonts-top" href="#">My Page</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-light fonts-top" href="#">Logout</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <nav class="navbar navbar-expand-md">
+            <div class="container-md">
+                <a id="home" class="navbar-brand d-none d-md-block" href="index.html">
+                    <img class="devfest-icon" src="img/DF22-Bevy-EventThumb@2x_togjfee.png"alt="devfest-icon"/>
+                </a>
+                <button class="navbar-toggler border border-0" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasNavbar" aria-controls="offcanvasNavbar" aria-label="toggler">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="offcanvas offcanvas-end" tabindex="-1" id="offcanvasNavbar" aria-labelledby="offcanvasNavbarLabel">
+                    <div class="offcanvas-header">
+                        <a class="offcanvas-title homeM" id="offcanvasNavbarLabel" data-bs-dismiss="offcanvas" aria-label="Close"><img class="devfest-icon" src="img/DF22-Bevy-EventThumb@2x_togjfee.png" alt="devfest-icon"/></a>
+                        <button type="button" class="btn-close" data-bs-dismiss="offcanvas" aria-label="Close" ></button>
+                    </div>
+                    <div class="offcanvas-body">
+                        <ul class="navbar-nav justify-content-end flex-grow-1 pe-3">
+                            <li class="nav-item p-2">
+                                <a class="nav-link active" data-bs-dismiss="offcanvas" aria-label="Close" aria-current="page" id="about-btn" href="index.html">About</a>
+                            </li>
+                            <li class="nav-item p-2">
+                                <a class="nav-link" href="#program-section">Program</a>
+                            </li>
+                            <li class="nav-item p-2">
+                                <a class="nav-link" href="#speakers">Speakers</a>
+                            </li>
+                            <li class="nav-item p-2">
+                                <a class="nav-link" href="#partners">Sponsers</a>
+                            </li>
+                            <li class="nav-item p-2">
+                                <a class="nav-link" href="https://gdg.community.dev/">GDG</a>
+                            </li>
+                            <li class="nav-item p-2">
+                                <a class="btn orange-font orange-border" href="https://gdg.community.dev/events/details/google-gdg-cairo-presents-devfest-cairo-2022/">Campain</a>
+                            </li>
+                        </ul>
```

Comment:
- [ ] As previously requested, in the about page mobile view, there seems to be no way of navigating to the home page. None of the links provided are for the home page and your users are not going to be able to navigate to the home page when they are in the about page mobile view. KIndly resolve this.

[Screencast from 06-04-2023 11:27:25 ASUBUHI.webm](https://user-images.githubusercontent.com/86318284/230319994-6d4e0e36-e4dd-4820-8c36-6a733be6bda1.webm)

Reply:
@Nemwel-Boniface  My bad.  I just got it.  Actually I thought you were  referring to another issue. But yeah let me fix it.

Reply:
@Nemwel-Boniface I removed a class that was causing this error. I fixed it already.